### DISCUSSION TITLE
TileSource level of detail

### DISCUSSION THREAD

**Discussion by Thomas-Lerman** [pelicanmapping/osgearth#2320] [Support Q&A]
We have a custom tile source layer that we put some high-resolution imagery. I should say we are on an older version of osgEarth. Even though we can limit the level of detail to the highest resolution, it appears the imagery is downsampled for display as one zooms out. The result is that the resolution of the imagery drops. Obviously, this is expected behavior beyond a point, but we can look at the original image in another non-tiling application at the same size and see better detail. I have not found anything that can be set, but wanted to double check to see if I have missed anything. Thank you for your help.

**Comment by gwaldron** [comment]
Thomas, A couple things you can try: you can adjust the `min_range_tile_factor` in your terrain options to a lower number (I believe the default is 7.0), or you can apply an OSG LOD Scale on the terrain to alter the LOD switching characteristics of the scene.
